File: SwipecardviewAndroid/app/src/main/java/velmm/com/swipecardviewandroid/ApiInterface.java
package velmm.com.swipecardviewandroid;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiInterface {

  @GET("5a8fefef3000004f00248c70")
  Call<List<Movie>> getAllMovies();
}

File: MaterialCardViewAndroid/app/src/main/java/com/example/materialcardviewandroid/ApiInterface.kt
package com.example.materialcardviewandroid

import retrofit2.Call
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory
import retrofit2.http.GET

interface ApiInterface {

    @GET("volley_array.json")
    fun getMovies() : Call<List<Movie>>

    companion object {

        var BASE_URL = "https://howtodoandroid.com/"

        fun create() : ApiInterface {

            val retrofit = Retrofit.Builder()
                .addConverterFactory(GsonConverterFactory.create())
                .baseUrl(BASE_URL)
                .build()
            return retrofit.create(ApiInterface::class.java)

        }
    }
}